planning/surround_obstacle_checker/src/node.cpp

```diff
@@ -289,22 +294,17 @@ void SurroundObstacleCheckerNode::onTimer()
   if (node_param_.pointcloud_enable_check && !pointcloud_ptr_) {
     RCLCPP_INFO_THROTTLE(
       this->get_logger(), *this->get_clock(), 5000 /* ms */, "waiting for pointcloud info...");
-    return;
```

Comment:
> I mean the surround obtacle checker also check obstacles based on pointcloud:
> 
> https://github.com/autowarefoundation/autoware.universe/blob/d99e56b3968d01233c0367a824dac154154beb9c/planning/surround_obstacle_checker/src/node.cpp#L397C35-L397C65
> 
> This part of the code still works even if all targets types are disabled (this is how I currenytly use the node).
> 
> With your PR, pointcloud based checking would not run anymore if all targets are disabled.

@VRichardJP Could you check latest file changes? I think pointcloud based checking can work even when all dynamic objects flags are `false`.